06_puppy/madhavansu/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+/**
```

Comment:
Godocs typically start with // FunctionName:
https://blog.golang.org/godoc-documenting-go-code - I'm surprised the linter didn't pick this up. I think your comments actually belong to the top of the file/ package level....